tools/rosbag/src/recorder.cpp

```diff
@@ -558,10 +559,19 @@ void Recorder::stopWriting() {
         {
             currently_recording_.erase(topic.first);
         }
-    }    
-    ROS_INFO("Closing '%s'.", target_filename_.c_str());
+    }
+    std::string target_filename = target_filename_;
```

Comment:
what if the file already exists?

Reply:
The old file will get overwritten with the new file